server/webhook.go

```diff
@@ -858,6 +883,19 @@ func (s *Server) createIssueFromPushEvent(ctx context.Context, pushEvent *vcs.Pu
 	return fmt.Sprintf("Created issue %q on adding %s", issue.Name, file), true, activityCreateList, nil
 }
 
+func getCommitsMessage(commitList []vcs.Commit) string {
+	switch len(commitList) {
+	case 0:
+		return ""
+	case 1:
+		return fmt.Sprintf("commit %s", commitList[0].ID[:7])
```

Comment:
Can we get the full commit ID and the full list below? What if ID has length < 7? Same below.